scripts/lib/generate-crds.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+ensure_crd_gen() {
+    local __crd_dir="$1"
+    CONTROLLER_GEN=$(which controller-gen)
+    if [[ -z "$CONTROLLER_GEN"  ]]
+    then
+      set -e
+      CONTROLLER_GEN_TMP_DIR="$(mktemp -d)"
+      cd "$CONTROLLER_GEN_TMP_DIR"
+      go mod init tmp
+      go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.3.0
+      rm -rf "$CONTROLLER_GEN_TMP_DIR"
+      CONTROLLER_GEN=$(GOBIN)/controller-gen
+    fi
+    $CONTROLLER_GEN "crd:trivialVersions=true" rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config="$__crd_dir"
```

Comment:
Remove the above line that actually calls the `controller-gen` binary.

The code that *calls* `controller-gen` should be in a `scripts/generate-crds.sh` file.

Also, the `paths="./..."` here should be a path to the `services/$SERVICE_ALIAS/apis/$API_VERSION/...` directory.

Additionally, remove "webook" and "rbac:roleName=manager-role" from the command. At this point, we are only generating the resource configuration manifests (CRDs)